src/components/Workflow/PlanAudioActions.tsx

```diff
@@ -61,7 +61,7 @@ export function PlanAudioActions(props: IProps) {
         <StyledIconButton
           id="passageShare"
           shared={mediaShared === IMediaShare.OldVersionOnly}
-          title={t.versions}
+          title={mediaShared ? 't.resourceEdit' : t.versions}
```

Comment:
should this be in quotes?

Reply:
I added this string and updated crowd in and the files based on it.